src/wp-includes/class-wp-term-query.php

```diff
@@ -748,56 +733,61 @@ public function get_terms() {
 		$cache_key    = "get_terms:$key:$last_changed";
```

Comment:
Sorry to be annoying and drop different things here each time I follow up. I intend well :) 

On the line prior, `$key` is limited to only use defined arguments -- IE you can't include `['john smith' => 'doctor who' ]` and affect the cache key.

There are now several defined arguments that have no affect on the first query for IDs:
* fields
* update_term_meta_cache

Should these be excluded from the cache key too?

Reply:
I had already thought of this! I was going to follow up with another ticket for this. There maybe other fields that we can ignore or tweak. But this feels out of scope of this ticket. `update_term_meta_cache` would have always been an issue.

Reply:
See follow up ticket. https://core.trac.wordpress.org/ticket/55352

Reply:
> I had already thought of this! I was going to follow up with another ticket for this.

That seems reasonable, thanks for posting the follow up.